app/grids/settings/recruitment/email_templates_grid.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Settings::Recruitment::EmailTemplatesGrid
```

Comment:
What do you think about adding a very simple test for this grid?

Reply:
in the internal project we have tests to check filters, but here no filters for this grid
I think I'll check success in controller test that page with grid is renders at all

Reply:
ok